### PR TITLE
Added scraper to get demographic data from ctp

### PR DESCRIPTION
**Pull request by sglyon**
closes CTP covid demographic data by state #65

Note that for now I am not gathering the testing data.

I looked through the underlying CTP google sheet and they are not consistent in the units from one state to the next.

For example, Illinois is reporting (and CTP collecting/reporting) pcr test specimens while Delaware is reporting pcr tests in units of unique people

If it is high priority we can work through this, but for now I skipped it

### CODE REVIEW

File: can_tools/bootstrap_data/covid_demographics.csv [hunk 1]
```diff
@@ -35,3 +35,5 @@ all,all,unknown,all
 all,all,all,male
 all,all,all,female
 all,all,all,other
+all,latinx,all,all
```
Comment (cc7768) [inline can_tools/bootstrap_data/covid_demographics.csv:38]: Looking at the actual CTP data, it seems like `latinx` lines up exactly with `ethnicity=hispanic`.

Is there an added bonus to having this?

File: can_tools/scrapers/ctp.py [hunk 2]
```diff
@@ -1,9 +1,133 @@
...
+            "Deaths_Multiracial": _deaths("multiple_other"),
+            "Deaths_Other": _deaths("other"),
+            "Deaths_Unknown": _deaths("unknown"),
+            "Deaths_Ethnicity_Hispanic": _deaths("all", "hispanic"),
+            "Deaths_Ethnicity_NonHispanic": _deaths("all", "non-hispanic"),
+            "Deaths_Ethnicity_Unknown": _deaths("all", "unknown"),
+            "Hosp_Total": _hosp("all"),
+            "Hosp_White": _hosp("white"),
+            "Hosp_Black": _hosp("black"),
+            "Hosp_LatinX": _hosp("latinx"),
```
Comment (cc7768) [inline can_tools/scrapers/ctp.py:84]: See comment on the bootstrap demographics file.

File: can_tools/scrapers/ctp.py [hunk 3]
```diff
@@ -1,9 +1,133 @@
...
+            "Cases_Multiracial": _cases("multiple_other"),
+            "Cases_Other": _cases("other"),
+            "Cases_Unknown": _cases("unknown"),
+            "Cases_Ethnicity_Hispanic": _cases("all", "hispanic"),
+            "Cases_Ethnicity_NonHispanic": _cases("all", "non-hispanic"),
+            "Cases_Ethnicity_Unknown": _cases("all", "unknown"),
+            "Deaths_Total": _deaths("all"),
+            "Deaths_White": _deaths("white"),
+            "Deaths_Black": _deaths("black"),
+            "Deaths_LatinX": _deaths("latinx"),
```
Comment (cc7768) [inline can_tools/scrapers/ctp.py:71]: See comment on the bootstrap demographics

File: can_tools/scrapers/ctp.py [hunk 4]
```diff
@@ -1,9 +1,133 @@
...
+                unit="beds",
+                race=race,
+                ethnicity=ethnicity,
+            )
+
+        column_map = {
+            "Cases_Total": _cases("all"),
+            "Cases_White": _cases("white"),
+            "Cases_Black": _cases("black"),
+            "Cases_LatinX": _cases("latinx"),
```
Comment (cc7768) [inline can_tools/scrapers/ctp.py:58]: See comment on the bootstrap demographics

**Review by cc7768 — COMMENTED** [review]
Overall this looks straightforward. My one question is what does keeping track of `latinx` buy us?

It looks like they're only reporting it when the sum of the `Cases_RACE` column (including `latinx`) sums to the total. I'm not too opinionated so I'm happy to keep it or leave it.

File: can_tools/bootstrap_data/covid_demographics.csv (hunk 1 above)
Comment (sglyon) [reply can_tools/bootstrap_data/covid_demographics.csv:38]: I just put it here to report the data as is. 

I think this is an "official" (from census or some other federal resource) classification for race and/or ethnicity. Mirroring the official is what I went after

**Review by cc7768 — APPROVED** [review]

**Pull request merged**